Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -19,14 +19,14 @@ import SwiftyJSON
 import Telemetry
 import Sentry
 
-private let KVOLoading = "loading"
-private let KVOEstimatedProgress = "estimatedProgress"
-private let KVOURL = "URL"
-private let KVOTitle = "title"
-private let KVOCanGoBack = "canGoBack"
-private let KVOCanGoForward = "canGoForward"
-private let KVOs = [KVOEstimatedProgress, KVOLoading, KVOCanGoBack, KVOCanGoForward, KVOURL, KVOTitle]
-private let KVOContentSize = "contentSize"
+private let KVOs = [
+    KVOConstants.estimatedProgress,
```

Comment:
for brevity this might work better
`private let KVOs: [KVOConstants] = [.estimatedProgress, .loading, .canGoBack, .canGoForward, .URL, .title]`